test/Directory.Build.props

```diff
@@ -48,6 +48,7 @@
     <Using Include="NSubstitute.ReturnsExtensions" />
     <Using Include="Xunit" />
     <Using Include="Xunit.Abstractions" />
+    <Using Condition="'$(TargetPlatformIdentifier)'==''" Include="VerifyXunit" />
```

Comment:
question: Is this required?

I thought it was redundant, because `Verify.Xunit` already ships with `ImplicitUsings`-support:
https://github.com/VerifyTests/Verify/blob/2f64cd19406244a38c34d8e8b737d936dde5e563/src/Verify.Xunit/buildTransitive/Verify.Xunit.props#L3-L7

Or is there a problem that I don't see?

Reply:
I got compiler errors without this:

<img width="991" height="700" alt="image" src="https://github.com/user-attachments/assets/27077252-d57c-4936-9d92-c438e3d06075" />

I'm not sure why... but adding this fixed it.